### DISCUSSION TITLE
How can I adjust Console width?

### DISCUSSION THREAD

**Discussion by brknsoul** [LykosAI/StabilityMatrix#411] [Q&A] [answered]
I think there was a variable you could set that would set the console width (ie: column size). I had to delete my settings.json, so I lost that setting.

**Comment by mohnjiles** [comment, marked as answer]
Hey, sorry for the delay 😅

If you still need it (or anyone else in the future does), it's the Environment Variable `COLUMNS`. Can set it to ~160 or so for a modern widescreen display.

**Reply by brknsoul** [reply]
Thanks for the answer. 

EDIT: I didn't realise that `COLUMNS` was used by more than just SM.
